fate/src/main/java/org/apache/accumulo/fate/ZooStore.java

```diff
@@ -458,4 +458,43 @@ public Serializable getProperty(long tid, String prop) {
       throw new RuntimeException(e);
     }
   }
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public List<ReadOnlyRepo<T>> getStack(long tid) {
+    String txpath = getTXPath(tid);
+
+    outer: while (true) {
+      List<String> ops;
+      try {
+        ops = zk.getChildren(txpath);
+      } catch (KeeperException.NoNodeException e) {
+        return null;
```

Comment:
Oh, I see you check for null down below. I think an empty list would be nicer (less of a chance for surprise), but would be happy with a javadoc update to `Store.getStack(long)` that says it might return null (since this isn't user-facing).